examples/common/src/maths.ads

```diff
@@ -43,11 +43,15 @@ package Maths is
       Transform  : out Singles.Matrix4);
    function Length (V : Singles.Vector3) return Single;
    function Minimum (I1, I2 : Int) return Int;
+   function Minimum (U1, U2 : UInt) return UInt;
    function Minimum (I1, I2 : Integer) return Integer;
-   function Minimum (I1, I2 : Single) return Single;
+   function Minimum (F1, F2 : Float) return Float;
+   function Minimum (S1, S2 : Single) return Single;
    function Maximum (I1, I2 : Int) return Int;
+   function Maximum (U1, U2 : UInt) return UInt;
    function Maximum (I1, I2 : Integer) return Integer;
-   function Maximum (I1, I2 : Single) return Single;
+   function Maximum (F1, F2 : Float) return Float;
+   function Maximum (S1, S2 : Single) return Single;
```

Comment:
I mostly oversaw or ignored this in previous PRs because it was a minor annoyance. But please, remove all the `Minimum` and `Maximum` functions, you do not need them. In Ada, every scalar type (including `range` types like `Integer` and `digits` types like `Float`) has [attributes `'Max` and `'Min`](http://www.adaic.org/resources/add_content/standards/05rm/html/RM-3-5.html#I1565) that do exactly the same thing as your functions. So please replace the usage of these functions with those attributes. Example: `Int'Max (23, 42)`.

Reply:
OK.
Seems a bit counterintuitive and non-standard programming but I'll comply with your wishes.

Reply:
There's a bit of reasoning behind it, mainly that these functions should be available for *all* (even user-defined) such types, therefore they cannot be normal subprograms defined in the Standard package.

I usually do not use the attributes for `'Pred` and `'Succ` on those types for incrementing / decrementing by one since that is actually more code than doing `+ 1`, but for `'Min` and `'Max`, I think we should use what the language provides us with instead of rolling our own.

Reply:
Fixed

Reply:
How did you fix it? The subprograms are still there.

Reply:
I'm not sure what happened; they've gone locally.
I must have forgotten to push it!
Should be gone now.